src/Core/SettingsChangesHistory.cpp

```diff
@@ -81,6 +81,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"iceberg_snapshot_id", 0, 0, "New setting."},
             {"parallel_replicas_for_cluster_engines", false, true, "New setting."},
             /// Release closed. Please use 25.4
+            {"enable_hdfs_pread", true, true, "Enable or disables pread for HDFS files. By default, `hdfsPread` is used. If disabled, `hdfsRead` and `hdfsSeek` will be used to read hdfs files."},
```

Comment:
Here's a comment that it's a new setting is sufficient. It's not a big deal, but please change it, since you have to resolve conflicts anyway

Reply:
Also, you should add this to the section above for version `25.4`, since `25.3` was already released

Reply:
done